nservicebus/security/encryption.md

```diff
@@ -96,12 +108,19 @@ This allows you to explicitly handled the encryption and decryption of each valu
 
 ## Message Encryption
 
-Message encryption leverages the pipeline to apply encryption to the whole message body.
+Message encryption encrypts the complete message body. It can even include the message headers if these would contain sensitive data.
 
-Once way of achieving this is using a `IMutateTransportMessages`.
+We currently do not offer this feature but it can be achieved by creating a transport message mutator using `IMutateTransportMessages`
 
 <!-- import MessageBodyEncryptor -->
 
-The this class can be then injected into the container using the following
+This class can be then injected into the container using the following:
 
 <!-- import UsingMessageBodyEncryptor -->
+
+
+## Transport Encryption
+
+However, using message encryption is often used to prevent intercepting data during transport. If this is the requirement then you can maybe on the transport for encryption. RabbitMQ, SQL Server, Azure Storage Queues and Azure Service Bus support SSL.
+
```

Comment:
So we dont support native msmq encryption? https://msdn.microsoft.com/en-us/library/ms699845(v=vs.85).aspx